GetMessage/handler.ts

```diff
@@ -117,6 +127,47 @@ type IGetMessageHandler = (
 const LegalMessagePattern = t.interface({ legal_data: LegalData });
 type LegalMessagePattern = t.TypeOf<typeof LegalMessagePattern>;
 
+/**
+ * Checks whether the client service can read advanced message info (read_status and payment_Status)
+ */
+export const canReadAdvancedMessageInfo = (
+  message:
+    | ExternalCreatedMessageWithoutContent
+    | ExternalCreatedMessageWithoutContent,
+  authGroups: IAzureApiAuthorization["groups"]
+): boolean =>
+  message.feature_level_type === FeatureLevelTypeEnum.ADVANCED &&
+  authGroups.has(UserGroup.ApiMessageReadAdvanced);
+
+/**
+ * Checks whether the client service can read message read status
+ *
+ * @param serviceId the subscription id of the service
+ * @returns false if user revoked the permission to access the read status, true otherwise
+ */
+export const canReadMessageReadStatus = (_serviceId: ServiceId): boolean =>
+  false;
```

Comment:
maybe we could add a TODO comment here: something like "waiting for opt-out definition on profiles domain"

Reply:
You're right, done in [20fae5a](https://github.com/pagopa/io-functions-services/pull/202/commits/20fae5a64590785abc8512c76fda4f681bbab510)